systemtest/gencorpora/catbulk.go

```diff
@@ -139,9 +210,9 @@ func handleReq(writer io.Writer) http.HandlerFunc {
 	})
 }
 
-func getWriter(writePath string) (io.WriteCloser, error) {
-	if writePath == "" {
+func getWriter(path string) (io.WriteCloser, error) {
+	if path == "" {
 		return os.Stdout, nil
```

Comment:
Ah, good point. Not sure if `NopCloser` for a writer is a good idea. What do you think about dropping `stdout` output and making the `directory` a required property?

Reply:
That sounds fine to me, it probably doesn't help to write the two different files to stdout anyway.

Reply:
Agreed. I have pushed a commit to remove `stdout` and now the write defaults to the current directory (didn't implement required on `write-dir` since it is not supported by default for the `flag` package and introduces additional logic for explicit validation in the code path).